src/Integrations/Integrations/Laravel/LaravelIntegration.php

```diff
@@ -52,6 +52,8 @@ public function init()
             return Integration::NOT_LOADED;
         }
 
+        $rootSpan->meta[Tag::COMPONENT] = LaravelIntegration::NAME;
```

Comment:
Can we put that into the specific trace/hook_method() calls which also assign the $rootSpan->name etc.?
Laravel is one of the few always-loaded integrations, as such, this shall not have a side effect by mere loading.

Reply:
Done. This also fixed all the side-affects within the tests that were adding the laravel component to spans that should not have had the component tag set, such as the Symfony console test